docs/Node Operators/configuration-variables.md

```diff
@@ -2,161 +2,605 @@
 layout: nodes.liquid
 section: nodeOperator
 date: Last Modified
-title: "Configuration Variables"
+title: "Configuring Chainlink Nodes"
 permalink: "docs/configuration-variables/"
-whatsnext: {"Enabling HTTPS Connections":"/docs/enabling-https-connections/"}
+whatsnext: {"Configuration UI":"/docs/configuration-ui/"}
```

Comment:
Shall we remove this until we fully document the Configuration UI? This is a broken URL right now.

Reply:
Removed.